webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -118,29 +129,31 @@ export default class CreateIssueModal extends PureComponent {
     handleIssueDescriptionChange = (issueDescription) => this.setState({issueDescription});
 
     renderIssueAttributeSelectors = () => {
-        if (!this.state.repo || (this.state.repo.permissions && !this.state.repo.permissions.push)) {
-            return null;
-        }
+        const otherOptionsDisabled = !this.state.repo || (this.state.repo.permissions && !this.state.repo.permissions.push);
+        const repoName = this.state.repo ? this.state.repo.name : '';
```

Comment:
Do you need to have `push` permissions in order to create an issue in a repository? cc @dipak-demansol